docs/secure-connections/ipsec-vpn.md

```diff
@@ -11,12 +11,18 @@ export const Highlight = ({children, color}) => ( <span style={{
       padding: '0.2rem',
     }}>{children}</span> );
 
+<p><span className="sauceGreen">Enterprise Only</span></p>
+
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 Internet Protocol Security VPN (IPSec VPN) is a technology that connects two private networks securely over the public internet. If your organization has firewall rules that limit your ability to run tests on Sauce Labs, you can use Sauce IPSec Proxy to connect to Sauce Labs without exposing your organization's IT infrastructure to security risks.
 
 Sauce IPSec Proxy is an enterprise-grade IPSec VPN solution that provides a secure, static connection between apps hosted on your private network (or local machine) and devices on the Sauce Labs cloud.
 
+:::note Add-on feature
```

Comment:
Hi @steilerDev ~ Can you please review this note? Any suggestions / corrections are appreciated - thank you!

Reply:
Looks good to me, thanks!